fourkitchens/scrummy
# Webpack/Babel setup

_Please ignore the branch names.. Instead of having a `rewrite` branch we merge into, we can just use this 4K fork `master` as the new frontend_
## Test
- Make sure `npm run build` and `npm start` do what they're supposed to.
- Browse to http://localhost:8080/webpack-dev-server/ and make sure breakpoint mixin is working by adjusting viewport.
